### PR TITLE
XW-206 | Additional check to prevent loading of any content; fix typo in i18n

### PR DESCRIPTION
**Pull request by vforge**
/cc @rogatty @Warkot @bognix 

Tested, works

### CODE REVIEW

**Comment by rogatty** [comment]
LGTM

File: extensions/wikia/GoogleFormTag/GoogleFormTagController.class.php [hunk 1]
```diff
@@ -33,7 +35,10 @@ public static function onParserFirstCallInit( Parser $parser ) {
 	public function renderTag( $input, array $args, Parser $parser, PPFrame $frame ) {
 		$sourceUrl = $this->cleanupUrl( $args['url'] );
 
-		if ( empty( $sourceUrl ) ) {
+		if (
+			empty( $sourceUrl ) ||
+			substr( $sourceUrl, 0, strlen( self::GOOGLE_FORM_BASE_URL ) ) !== self::GOOGLE_FORM_BASE_URL
+			) {
```
Comment (bognix) [inline extensions/wikia/GoogleFormTag/GoogleFormTagController.class.php]: indentation

**Comment by bognix** [comment]
LGTM

File: extensions/wikia/GoogleFormTag/GoogleFormTagController.class.php (hunk 1 above)
Comment (Warkot) [inline extensions/wikia/GoogleFormTag/GoogleFormTagController.class.php]: Indeed.

**Comment by Warkot** [comment]
LGTM

**Pull request merged**